### PR TITLE
DOC: clarify handling of conditions

### PR DESCRIPTION
**Pull request by jjnurminen**
Addresses #3086

### CODE REVIEW

File: mne/epochs.py [hunk 1]
```diff
@@ -696,6 +696,12 @@ def average(self, picks=None):
         -------
         evoked : instance of Evoked
             The averaged epochs.
+            
+        Notes
+        -----
+        Computes an average of ALL epochs in the instance, even if
```
Comment (jona-sassenhagen) [inline mne/epochs.py]: I don't like the all caps, just make it "all".

File: mne/evoked.py [hunk 2]
```diff
@@ -116,6 +116,11 @@ def save(self, fname):
         ----------
         fname : string
             Name of the file where to save the data.
+            
+        Notes
+        -----
+        An evoked object contains a single condition only. To write
```
Comment (jona-sassenhagen) [inline mne/evoked.py]: I think the first sentence is superfluous.

Comment (jjnurminen) [inline mne/evoked.py]: What would be a good place to clarify that Evoked is single-condition, even though Epochs is not (and maybe add the dict comprehension for averaging multiple categories)? This confused me for awhile.
Should I put it into the Notes for the Epochs/Evoked classes?

Comment (jjnurminen) [inline mne/evoked.py]: pushed proposal as a new commit

File: mne/epochs.py [hunk 3]
```diff
@@ -696,6 +696,12 @@ def average(self, picks=None):
         -------
         evoked : instance of Evoked
             The averaged epochs.
+            
+        Notes
+        -----
+        Computes an average of all epochs in the instance, even if
+        they correspond to different conditions. To average by condition,
+        do epochs[condition].average() for each condition separately.
```
Comment (Eric89GXL) [inline mne/epochs.py]: double-backticks on code to render properly

File: mne/epochs.py [hunk 4]
```diff
@@ -1986,6 +1992,7 @@ class Epochs(_BaseEpochs):
         list tag (e.g. ['audio', 'left'] selects 'audio/left' and
         'audio/center/left', but not 'audio/right').
 
+
```
Comment (Eric89GXL) [inline mne/epochs.py]: don't need

File: mne/evoked.py [hunk 5]
```diff
@@ -116,6 +122,11 @@ def save(self, fname):
         ----------
         fname : string
             Name of the file where to save the data.
+            
+        Notes
+        -----
+        To write multiple conditions into a single file, use 
+        mne.write_evokeds().
```
Comment (Eric89GXL) [inline mne/evoked.py]: use :func:\`mne.write_evokeds\`

**Comment by Eric89GXL** [comment]
LGTM, @drammock WDYT?

File: mne/evoked.py [hunk 6]
```diff
@@ -92,6 +92,12 @@ class Evoked(ProjMixin, ContainsMixin, UpdateChannelsMixin,
         Evoked response.
     verbose : bool, str, int, or None.
         See above.
+        
+    Notes
+    -----
+
```
Comment (drammock) [inline mne/evoked.py]: Delete extra line here

**Comment by drammock** [comment]
LGTM. Thanks @jjnurminen

File: mne/evoked.py [hunk 7]
```diff
@@ -116,6 +121,11 @@ def save(self, fname):
         ----------
         fname : string
             Name of the file where to save the data.
+            
+        Notes
+        -----
+        To write multiple conditions into a single file, use 
+        :func: `mne.write_evokeds()`.
```
Comment (agramfort) [inline mne/evoked.py]: this does not render fine. See

https://circle-artifacts.com/gh/mne-tools/mne-python/663/artifacts/0/home/ubuntu/mne-python/doc/_build/html/generated/mne.Evoked.html#mne.Evoked

Comment (Eric89GXL) [inline mne/evoked.py]: remove space and parens

Comment (jjnurminen) [inline mne/evoked.py]: Sorry for rushing a bit and not RTFM. I fixed it, but now AppVeyor fails for some reason that seems to be unrelated to the diff.

**Comment by Eric89GXL** [comment]
Check Travis

https://travis-ci.org/mne-tools/mne-python/jobs/122300815#L2819

**Comment by agramfort** [comment]
thx @jjnurminen

**Pull request merged**